structlog/stdlib.py

```diff
@@ -141,6 +231,35 @@ def __call__(self, *args):
         return logging.getLogger(name)
 
 
+class PositionalArgumentsFormatter(object):
+    """
+    Applies stdlib-like string formatting to the `event` key with the arguments
+    in the `positional_args` key. This is populated by
+    `structlog.stdlib.BoundLogger` or can be manually set.
+
+    `positional_args` can be any iterable, but a dictionary as the single
```

Comment:
I think this `any iterable` isn’t true anymore, now we removed the call to `tuple()`? I’m also not sure what the rest of the comment means, do you? :confused: